mayastor/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -372,6 +375,60 @@ impl Drop for Nexus {
     }
 }
 
+extern "C" fn nvmf_create_snapshot_hdlr(req: *mut spdk_nvmf_request) -> i32 {
```

Comment:
Im not sure this is the right place for this code, maybe the nvmf subsystem bits are more appropiated for this, but we can move it around as it nears completion.